.github/workflows/portal-checks.yml

```diff
@@ -22,10 +22,10 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Set up Python 3.9
+      - name: Set up Python 3.11
```

Comment:
is this actually needed? the make task will configure a docker-in-docker container anyway with correct python ver & poetry init. the Run Tests task doesn't have these steps

Reply:
Would you mind removing now?